packages/teams-js/src/public/runtime.ts

```diff
@@ -111,6 +113,7 @@ export const teamsRuntimeConfig: IRuntime = {
       fullTrust: {},
     },
     remoteCamera: {},
+    search: {},
```

Comment:
You probably want to omit this. This is the list of what the default capabilities are in teams clients that haven't been upgraded yet. I don't believe the search capability will work on old teams clients because it is new, so omitting this will make sure search.isSupported will return false in those scenarios.